Makefile.frag

```diff
@@ -88,3 +88,27 @@ generate-function-map: all
 	else \
 		echo "ERROR: Cannot generate function maps without CLI sapi."; \
 	fi
+
+test-no-build:
```

Comment:
> Alternatively, we can also extract the call to run-tests.php

I was originally going to suggest replacing this Makefile target with additional logic in `.evergreen/run-tests.sh`, but seeing how messy this is (and integrated with the rest of the Makefile) I think this is the right call. 

> at that point we may as well consider using a different test runner entirely, such as PHPUnit (which can run phpt files as well).

I think this is totally worth exploring. Would you mind creating a ticket so we can revisit the idea?

I would never consider using PHPUnit with PHPC for local development (since it'd likely complicate things like backtraces and leak detection), but it may actually make sense for CI assuming it handles all of the same syntax of `run-tests.php` and generates comparable test suite output. And we'd also be able to test it alongside `run-tests.php` (e.g. introduce PHPUnit in just an extra task to start). I assume it will also enable better integration with Evergreen, such that we could start populating the "Tests" tab like we do for PHPLIB.

Reply:
@alcaeus: :point_up:

Reply:
Created PHPC-2306 to evaluate PHPUnit as a test runner for PHPC. I also created PHPC-2307 to track attaching test results separately.